src/libs/ReportUtils.ts

```diff
@@ -8544,7 +8544,8 @@ function canRequestMoney(report: OnyxEntry<Report>, policy: OnyxEntry<Policy>, o
     }
 
     if (isMoneyRequestReport(report)) {
-        return canAddTransaction(report);
+        const canAddTransactions = canAddTransaction(report);
+        return isExpenseReport(report) ? isOwnPolicyExpenseChat && canAddTransactions : canAddTransactions;
```

Comment:
Feels like this logic should already be captured in the `canAddTransaction` method as its the lower level one. Why do we move it up here? then if someone will use the canAddTransaction elsewhere this bug might reappear